mycore-base/src/main/java/org/mycore/access/MCRAccessCacheManager.java

```diff
@@ -96,4 +105,28 @@ public void removePermission(String id, String permission) {
         permissionCache.remove(handle);
     }
 
+    public void removePermission(String... ids) {
+        MCRCache<MCRPermissionHandle, Boolean> permissionCache = accessCache.get();
+        removePermissionFromCache(permissionCache, ids);
+    }
+
+    private void removePermissionFromCache(MCRCache<MCRPermissionHandle, Boolean> permissionCache, String... ids) {
```

Comment:
use `Set<String>` instead of `String...` for ids, 
this would save you from having to create the set every time the method is called again.